### PR TITLE
Namespaced protocol buffers

### PR DESCRIPTION
**Pull request by maciekrb**
**WARNING** This is a backwards incompatible release. 

Changes include: 
- Protocol buffers are now namespaced under `matrix_io.*`
- MALOS protos are now found under `matrix_io.malos.v1`
- MALOS protos broken down into smaller units `comm`, `io`, `sense`
- MALOS ZWave protos added
- Python proto package resources and docs (now moved to a separate repo on bitbucket)
- NodeJS proto package resources and docs (now moved to a separate repo on bitbucket)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Change Log
+## [Unreleased]
+## [0.0.14] - 2017-08-03
```
Comment (hpsaturn) [inline CHANGELOG.md]: maybe we need to sync this changelog with debian changelog? or should we maintain these separated? I'd rather the first option.

**Comment by eighteyes** [comment]
@maciekrb we still have to migrate over in matrix land.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- AreaConfig, AreaZndZonde Config proto merge into namespaced version
+- debian package configuration
+
+## [0.0.10] - 2017-02-17
+### Added
+- Recognition service protos
+- Vision service protos (demographics services)
+- CMake now auto-discovers protos recursively
+
+
```
Comment (di3goleite) [inline CHANGELOG.md]: Please, remove this empty line

**Comment by maciekrb** [comment]
@eighteyes let me know if you guys need help with that migration. I want to make sure we cover all our bases.

**Comment by arhuaco** [comment]
ACK.

**Comment by maciekrb** [comment]
Namespaced protocol buffers finally going mainstream

**Pull request merged**